_data/events.yml

```diff
@@ -0,0 +1,10 @@
+- name: "PEARC19"
+  location: Chicago, IL
+  url: https://www.pearc19.pearc.org/
+  expires: 2019-08-01
+  description: Join us at PEARC19 (https://www.pearc19.pearc.org/) for a Birds of a Feather (BOF) session "Building a Community of Research Software Engineers."  Our session is scheduled for 5:15 PM on Monday, July 29.
+- name: "Supercomputing 2019 (SC19)"
+  location: Denver, CO
+  url: https://sc19.supercomputing.org/
+  expires: 2019-11-22
+  description: "Join us at SC19 for two RSE related panel sessions:<br>  - 1. [Developing and Managing Research Software in Universities and National Labs](https://sc19.supercomputing.org/presentation/?id=pan108&sess=sess226) <br>  - 2. [Sustainability of HPC Research Computing - The Importance of Collaborations  for Fostering Career Paths for Facilitators, Research Software Engineers, and Gateway Creators](https://sc19.supercomputing.org/presentation/?id=pan109&sess=sess227) <br><br>See the [SC panel schedule](https://sc19.supercomputing.org/program/panels/#schedule) for timing and location."
```

Comment:
Totally minor thing, but can you remove the "-" before the numbered items.  I

Reply:
It doesn't really look like a list when I do that :/

![image](https://user-images.githubusercontent.com/814322/62576652-48913c80-b86b-11e9-8277-d238124995a1.png)